src/SprykerEco/Shared/AfterPay/Transfer/after-pay.transfer.xml

```diff
@@ -31,238 +31,232 @@
     </transfer>
 
     <transfer name="AfterPayAuthorizeRequest">
-        <property name="payment" type="AfterPayRequestPayment" />
-        <property name="customer" type="AfterPayRequestCustomer" />
-        <property name="order" type="AfterPayRequestOrder" />
-        <property name="checkoutId" type="string" />
+        <property name="payment" type="AfterPayRequestPayment"/>
+        <property name="customer" type="AfterPayRequestCustomer"/>
+        <property name="order" type="AfterPayRequestOrder"/>
+        <property name="checkoutId" type="string"/>
     </transfer>
 
     <transfer name="AfterPayValidateBankAccountRequest">
-        <property name="bankAccount" type="string" />
-        <property name="bankCode" type="string" />
-        <property name="bankNumber" type="string" />
+        <property name="bankAccount" type="string"/>
+        <property name="bankCode" type="string"/>
+        <property name="bankNumber" type="string"/>
     </transfer>
 
     <transfer name="AfterPayValidateBankAccountResponse">
-        <property name="isValid" type="bool" />
-        <property name="riskCheckMessages" type="AfterPayRiskCheckMessage[]" singular="riskCheckMessage" />
+        <property name="isValid" type="bool"/>
+        <property name="riskCheckMessages" type="AfterPayRiskCheckMessage[]" singular="riskCheckMessage"/>
     </transfer>
 
     <transfer name="AfterPayRiskCheckMessage">
-        <property name="type" type="string" />
-        <property name="code" type="string" />
-        <property name="message" type="string" />
-        <property name="customerFacingMessage" type="string" />
-        <property name="actionCode" type="string" />
-        <property name="fieldReference" type="string" />
-    </transfer>
-
-    <transfer name="AfterPayValidateBankAccountRequest">
-        <property name="bankAccount" type="string" />
-        <property name="bankCode" type="string" />
-        <property name="bankNumber" type="string" />
```

Comment:
removed? it's BC break. So if it is not used - just leave it as is.

Reply:
Not removed, only deleted whitespaces before "/>"

Reply:
ok. it's moved to another line.